Forta-Bots-Challenge-1/README.md

```diff
@@ -0,0 +1,30 @@
+# Nethermind Update and Creation Detection Bot
+
+## Description
+
+This bot detects created and updated bots deployed by Nethermind to the Forta Bot Registry
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- FORTA-1
+  - Fired when a new bot has been deployed to the Forta Bot Registry by Nethermind
+  - Severity is always set to "low" (mention any conditions where it could be something else)
+  - Type is always set to "info" (mention any conditions where it could be something else)
+  - agentID, address deployed from, call metadata and chainIDs are stored
```

Comment:
Please take a look at how we format the `metadata` portion of the alerts. I.e. with parent-child bullet points.